specification/maps/data-plane/Route/stable/2025-01-01/route.json

```
@@ -0,0 +1,3783 @@
+        },
+        "geometry": {
+          "$ref": "#/definitions/GeoJsonMultiPoint"
+        },
+        "properties": {
+          "description": "MultiPoint feature properties object which specifies the origin features and destination features for the input matrix.",
+          "type": "object",
+          "$ref": "#/definitions/InputRouteMatrixProperties"
+        }
+      },
+      "required": [
+        "type",
+        "geometry",
+        "properties"
+      ]
+    },
+    "InputRouteMatrixProperties": {
+      "type": "object",
+      "description": "Specifies the properties object for the input matrix.",
+      "properties": {
+        "pointType": {
+          "description": "Specifies the origin MultiPoint type and destination MultiPoint type for the input matrix.",
+          "type": "string",
+          "enum": [
+            "origins",
+            "destinations"
+          ],
+          "x-ms-enum": {
+            "name": "RouteMatrixTypeEnum",
+            "modelAsString": true,
+            "values": [
+              {
+                "value": "origins",
+                "description": "MultiPoint features that define the origin locations in the input matrix."
+              },
+              {
+                "value": "destinations",
+                "description": "MultiPoint features that define the destination locations in the input matrix."
+              }
+            ]
+          }
+        }
+      }
+    },
+    "RouteMatrixFeatureProperties": {
+      "description": "Route Matrix properties.",
+      "type": "object",
+      "properties": {
+        "summary": {
+          "$ref": "#/definitions/RouteMatrixSummary"
+        },
+        "matrix": {
+          "description": "The matrix of route results.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/RouteMatrixItemResult"
+          }
+        }
+      }
+    },
+    "RouteMatrixItemResult": {
+      "type": "object",
+      "description": "Route Matrix item result",
+      "properties": {
+        "statusCode": {
+          "description": "The HTTP status code of the current cell.",
+          "type": "integer",
+          "format": "int32"
+        },
+        "originIndex": {
+          "description": "Index of the origin point",
+          "type": "integer",
+          "format": "int32"
+        },
+        "destinationIndex": {
+          "description": "Index of the destination point",
+          "type": "integer",
+          "format": "int32"
+        },
+        "departureTime": {
+          "description": "The estimated departure time, which takes into account the traffic conditions, is formatted as a `dateTime` value defined by [RFC 3339, section 5.6](https://www.rfc-editor.org/rfc/rfc3339#section-5.6). It will reference the timezone offset by either `departAt` or `arrivalAt`. If not, then the UTC time will be used. If departAt or arriveAt is `any`, then departureTime is absent.",
+          "format": "date-time",
```

Property should have a defined type. 
```suggestion
          "type": "string",
          "format": "date-time",
```